mmjstool/src/i18n_extract.js

```diff
@@ -97,6 +95,35 @@ function extractFromFile(path) {
             } else if (node.callee.name === 't') {
                 const id = node.arguments[0] && node.arguments[0].value;
                 translations[id] = '';
+            } else if ((node.callee.type === 'MemberExpression' && node.callee.property.name === 'defineMessages') ||
+                node.callee.name === 'defineMessages') {
+                if (node.arguments && node.arguments[0] && node.arguments[0].properties && node.arguments[0].properties.length !== 0) {
+                    for (const property of node.arguments[0].properties) {
+                        if (property.type && property.type === 'Property' && property.key && property.key.name !== '' && property.value &&
+                            property.value.type === 'ObjectExpression' && property.value.properties && property.value.properties.length !== 0) {
+                            const idProperty = property.value.properties[0];
```

Comment:
why are we assuming that the first property is the `id`? same with line 132

I like it better as it was done in the previous if block where the key name value is being check

Reply:
that makes sense